src/core/apim_checkout.tf

```diff
@@ -348,6 +348,15 @@ resource "azurerm_api_management_api_operation_policy" "transaction_authorizatio
   )
 }
 
+resource "azurerm_api_management_api_operation_policy" "new_transaction_origin" {
```

Comment:
this policy should be extended to all ecommerce API for checkout

Reply:
So can I move this policy to the base one?

Reply:
You can delete `azurerm_api_management_api_operation_policy.new_transaction_origin` and add 
```xml
    <set-header name="X-Client-Id" exists-action="override" >
      <value>CHECKOUT</value>
    </set-header>
```
in
`src/domains/ecommerce-app/api/ecommerce-transactions-service/v1/_base_policy.xml.tpl`

Reply:
fixed in https://github.com/pagopa/pagopa-infra/pull/607/commits/6e372723ccde22af54cf8182cc3ed44bac0138ab